src/rest-server/docs/swagger.yaml

```diff
@@ -278,38 +278,258 @@ paths:
+        '404':
+          $ref: '#/components/responses/NoUserError'
+        '500':
+          $ref: '#/components/responses/UnknownError'
+  /api/v2/users/me:
     put:
       tags:
         - user
-      summary: FIXME
+      summary: Update user's own profile.
+      description: >-
+        Update user's own profile in the system.
+        Only allowed in basic mode
+      operationId: updateUserSelf
+      security:
+        - bearerAuth: []
+      requestBody:
+        description: Update user's own profile
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                data:
+                  type: object
+                  required:
+                    - username
+                  properties:
+                    username:
+                      type: string
+                      description: user name
+                    email:
+                      type: string
+                      description: >-
+                        Email address.
+                        When patch is false, it's required.
+                        When patch is true, it's optional.
+                    newPassword:
+                      type: string
+                      description: >-
+                        New password.
+                        When patch is false, it's required.
+                        When patch is true, it's optional.
+                    oldPassword:
+                      type: string
+                      description: >-
+                        Old password.
+                        When newPassword exists, it's required.
+                        When newPassword doesn't exists, it's useless.
+                patch:
+                  type: Boolean
+                  default: false
+      responses:
+        '201':
+          description: Succeeded
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Response'
+              example:
+                message: 'update group {username} successfully.'
+        '401':
+          $ref: '#/components/responses/UnauthorizedUserError'
+        '403':
+          $ref: '#/components/responses/ForbiddenUserError'
+        '404':
+          $ref: '#/components/responses/NoUserError'
+        '500':
+          $ref: '#/components/responses/UnknownError'
+  '/api/v2/users/{user}':
+    parameters:
+      - $ref: '#/components/parameters/user'
+    get:
+      tags:
+        - user
+      summary: Get an user's data.
+      description: >-
+        Get an user's data.
+        Only allowed in basic mode
+      operationId: getUser
+      security:
+        - bearerAuth: []
       responses:
         '200':
           description: Succeeded
-          content: {}
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
```

Comment:
this is the same to the item type in `get /api/v2/users`, right?

Reply:
yes